cmd/jaeger/README.md

```diff
@@ -1,10 +1,8 @@
 # jaeger
 
-This is experimental Jaeger V2 based on OpenTelemetry collector.
+Jaeger V2 based on OpenTelemetry collector.
```

Comment:
should we link the new blog post here?

Reply:
The other post is more technical. Ideally this should point to the docs instead of a blog post.

Reply:
do we want to do that in this PR?

Reply:
It's not a particularly critical README. We do need to push more visibility though for v2, maybe in the main README.